### ISSUE TITLE
Aircraft MD-500E (North Korea - 1985) 87 Less Attrition Anti tank and SF insertion?

### ISSUE THREAD

**Issue by KLABDB3K** [PygmalionOfCyprus/cmo-db-requests#3864]
### DB Selector

DB3K

### DBID to Copy

Aircraft #5558 - MD-500E (El Salvador - 1984)

### Hypothetical

No

### Name

MD-500E (North Korea - 1985) 87 Less Attrition Anti Tank / SOF

### Country

North Korea

### Service

Air Force

### In Commission

1985

### Number in Service

87

### Sensors

Mk 1 Eyeball. 
Guidance unit for later Bulsae-3 or Bulsae-5?

### Mounts

Nil. 

### Loadout Options

4 x AT-3 Sagger (Susong-Po antitank missiles), 2 per side in over under configuration. 

4 x AT-4 Spigot (Bulsae-2 antitank missiles), 2 per side in over under configuration. 

Later versions of the Susong-Po missile may well be RF guided SACLOS not wire but are not modelled and the later version of the 

Use of the Bulsae-3? (modified laser command guided Konkurs/Spandrel) or Bulsae-5 (Kornet/Spriggan) could not be ruled out but would be a later version circa 2012 and would require the laser sight unit.

4 x SF or Saboteur

4 x Troops

Unarmed Recon

There is no pictorial evidence as such but a automatic grenade launcher (AGS-17 based?) has also reportedly been mounted under the fuselage forward firing. 

### Other Changes

With the low level flight terrain modelling this will make an interesting addition.

### Sources

Armed Forces of North Korea - Path of Songun.
Mitzer and Oliemans.
ISBN 978-1-910777-14-5 
pp.120 Graphic.
pp.130 - describes mixed 9M111 and 9M14 missile armed MD-500 and ventral 30mm ? AGL pod. 

https://nationalinterest.org/blog/reboot/how-north-korea-got-its-hands-87-us-md-500-helicopters-160926
2013 parade footage -
https://www.youtube.com/watch?v=Pk1hQCNwtrU 
https://youtu.be/Pk1hQCNwtrU?t=5806 MD-500E with Sagger pylons. 

https://www.youtube.com/watch?v=B9ogY9Xe_3I Four in formation, 

https://armyrecognition.com/weapons_defence_industry_military_technology_uk/north_korea_has_upgraded_old_anti-tank_missile_with_laser_guided_system_called_bulsae-3.html Bulsae 3 laser SACLOS 9M111 AT-4 Spigot ? 
http://www.armscontrolwonk.com/archive/207370/oryx-blog-on-dprk-arms-exports/

**Comment by FrangibleCover** [comment]
Good catch, I didn't realise these weren't in the DB. Regrettably, there's no way of representing one of the most interesting hypothesised uses of these helicopters, which is painting them to resemble ROKA/ROKAF MD 500MD helicopters to allow them to more effectively infiltrate South Korea.

**Comment by KLABDB3K** [comment]
I was surprised they weren't in too. Did a scenario with MH6 and US SOF and the little Birds at night at ultra low level are a real giant slayer.

**Comment by KLABDB3K** [comment]
https://youtu.be/OJcX8ha2QCM?si=uwle6z4OMbl_58Bu
would still like this to be in the DB3K please?

**Comment by KMPDB** [comment]
Added DB519.
